lib/perfdata/influxdbwriter.cpp

```diff
@@ -471,6 +467,8 @@ void InfluxdbWriter::Flush()
 	Log(LogDebug, "InfluxdbWriter")
 		<< "Flushing data buffer to InfluxDB.";
 
+	IncreaseQueryCount(m_DataBuffer.size());
+
```

Comment:
You’re using this method only once. Do you need it at all?

Reply:
How am I going to find out the values if I don't use the method?

Reply:
You just put the method's code here.

Reply:
And when we eventually have to use it twice?

Reply:
We won’t.